### DISCUSSION TITLE
Issue on enabling custom SMTP with AWS SES

### DISCUSSION THREAD

**Discussion by Carlchk** [supabase/supabase#2767] [Questions] [answered]
Hello, 

I have fully configured the SMTP service using the information provided by AWS SES. But this service didn't work and keep showing" Error sending confirmation mail" while handling user registration.  I have successfully sent a test email via AWS SES to my mailbox, and it actually works, and I have received an email from the AWS SES. So, I am sure that the AWS SES is working fine.

But when I set it in Supa, it got a problem. Does anyone have experience in configuring AWS SES and supabase custom SMTP config? or I have to update to the Pro then I can access the custom SMTP service. Thx
 
![Image of supabase config](https://cdn.discordapp.com/attachments/843999948717555735/873888960508743700/unknown.png)

**Comment by farzd** [comment]
not working for me either, have similar config with AWS SES 
not sure which port to use here, we are given the following options
```
SMTP endpoint
email-smtp.eu-west-2.amazonaws.com

STARTTLS Port
25, 587 or 2587

Transport Layer Security (TLS)
Required

TLS Wrapper Port
465 or 2465

```
Theres also policy configurations etc

**Comment by farzd** [comment, marked as answer]
so this is now working,  issue was to do with SES being in sandbox mode, you need to have your account processed to be production mode for you to be able to send emails via a verified email/domain  @Carlchk

**Reply by Carlchk** [reply]
Thanks. AWS SES has a strict usage restriction on using their service to be SMTP relay. I have changed to use SendGrid to be the SMTP relay. Thanks for providing a solution for this issue.
